assets/slides.css

```diff
@@ -10,4 +10,8 @@ body {
 	background-position: center;
 	background-repeat: no-repeat;
 	background-size: cover;
-}
\ No newline at end of file
+}
+
+.content {
+  max-width: 700px;
```

Comment:
Px or Em? What does this mean for high resolution or low resolution screens?

Reply:
Ah sorry, to give answer to your actual question: It might look a bit narrow on high resolutions. Will add a follow up ticket (see above): `Fintetune styles consider resolutions`